src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -48,60 +49,99 @@ internal static SafeChannelBindingHandle QueryChannelBinding(SafeSslHandle conte
 
         internal static SafeSslHandle AllocateSslContext(SslProtocols protocols, SafeX509Handle certHandle, SafeEvpPKeyHandle certKeyHandle, EncryptionPolicy policy, bool isServer, bool remoteCertRequired)
         {
-            SafeSslHandle context = null;
+            bool hasCertReference = false, 
+                hasCertKeyReference = false;
```

Comment:
Nit: These should either be on the same line, e.g.
```C#
bool hasCertReference = false, hasCertKeyReference = false;
```
or should each have an explicit type, e.g.
```C#
bool hasCertReference = false;
bool hasCertKeyReference = false;
```